game.js

```
@@ -1840,3 +1852,874 @@ function resolveWallCollisions() {
+      break;
+      
+    case 'autoAttack':
+      // Toggle auto attack (equivalent to 'e' key)
+      toggleAutoAttack();
+      break;
+      
+    case 'apple':
+      // Quick select apple (equivalent to 'q' key)
+      const appleSlot = myPlayer?.inventory?.slots.findIndex(
+        (item) => item?.id === "apple"
+      );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)